reactApp/app1.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Child from './child.jsx'
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {temp: "", name: "", array: []};
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleButton = this.handleButton.bind(this);
+  }
+  
+  handleChange(event) {
+    this.setState({temp: event.target.value});
+  }
+
+  handleButton(event) {
+    function check(element) {
+      return element == event.target.value;
+
+    }
+
+    if (!this.state.array.find(check)) { 
```

Comment:
@VARTIKASINGH1234 use arrow function instead. Don't just copy and paste the code.